### DISCUSSION TITLE
What is the state of LG webOS support?

### DISCUSSION THREAD

**Discussion by stamminator** [AvaloniaUI/Avalonia#15237] [Q&A] [answered]
I've found a few mentions online that Avalonia works on webOS ([here](https://twitter.com/AvaloniaUI/status/1687817554155204608) and [here](https://www.youtube.com/watch?v=THKjD-ZwGPk)), but strangely enough, not a single mention of webOS comes up when I do a repo-wide search here.

Can we get definitive word on what the state of webOS support is? Are there any special considerations when [packaging an app](https://webostv.developer.lge.com/develop/getting-started/web-app-types#basic-web-app)?

An addition to the Solitaire sample might be a good place to prove it out

**Comment by maxkatz6** [comment, marked as answer]
11.1 supports AndroidTV, AppleTV and TizenTV.
There is no proper support for WebOS at the moment.

But you can try Browser/WASM backend running as a webapp on webOS. Which probably wouldn't have a great performance.
